core/src/main/java/org/apache/calcite/rel/core/Sort.java

```diff
@@ -86,17 +126,7 @@ protected Sort(
       RelCollation collation,
       @Nullable RexNode offset,
       @Nullable RexNode fetch) {
-    super(cluster, traits, child);
```

Comment:
Thanks for the contribution, overall looks good, there are some compile failure, can you fix that ?